docs/rules/accessor-pairs.md

```diff
@@ -92,7 +92,7 @@ Object.defineProperty(o, 'c', {
 
 ```
 
-#### getWithoutSet
+#### `getWithoutSet`
```

Comment:
I think this is correct as-is. The quoted  values are for the quoted options. This one is an object property.